<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWidget</class>
 <widget class="QWidget" name="ProgressWidget">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>253</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background: transparent;
}

QLabel#l_title {
	font: 14 NotEurostile;
	font-weight: regular;
	color: #392518;
}

QPushButton#btn_close {

    border-image: url(:/img/btn/close_install.png);
    background-repeat: no-repeat;
}

QPushButton#btn_pause {
    border-image: url(:/img/btn/pause_install.png);
    background-repeat: no-repeat;
}

QWidget#ProgressWidget {
	background-image: url(&quot;:/img/progressbar_bg.png&quot;);
	background-repeat: no-repeat;
}

QProgressBar {
	background-color: transparent;
	border-radius: 0px;
}

QLabel#l_progressBar_bg {
	background-image: url(&quot;:/img/progressbar_empty.png&quot;);
	background-repeat: no-repeat;
}

QProgressBar#pb_primary::chunk {
    border-width: 1px;
    border-style: solid;
    border-radius: 2px;
    border-color: rgb(47, 180, 224);
	background-color: qlineargradient(spread:pad, x1:0.503, y1:0.818182, x2:0.503, y2:0.091, stop:0 rgba(11, 172, 224, 255), stop:0.886551 rgba(140, 224, 248, 255), stop:0.886964 rgba(222, 248, 255, 255), stop:1 rgba(222, 248, 255, 255));
}
 
QProgressBar::chunk_error {
    border-width: 1px;
    border-style: solid;
    border-radius: 3px;
    border-color: rgb(204, 76, 1);
    background-color: qlineargradient(spread:pad, x1:0.503, y1:0.818182, x2:0.503, y2:0.091, stop:0 rgba(237, 52, 8, 255), stop:0.886551 rgba(252, 132, 64, 255), stop:0.886964 rgba(246, 217, 153, 255), stop:1 rgba(246, 217, 153, 255));
}

QProgressBar#pb_secondary::chunk {
    border-width: 1px;
    border-style: solid;
    border-radius: 2px;
	border-color: rgb(206, 109, 0);
	background-color: qlineargradient(spread:pad, x1:0.503, y1:0.818182, x2:0.503, y2:0.091, stop:0 rgba(238, 132, 17, 255), stop:0.886551 rgba(253, 193, 64, 255), stop:0.886964 rgba(255, 249, 236, 255), stop:1 rgba(255, 249, 236, 255));
}


</string>
  </property>
  <widget class="QProgressBar" name="pb_primary">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>86</y>
     <width>386</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>386</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>386</width>
     <height>17</height>
    </size>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_status">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>115</y>
     <width>384</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>34%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>28</y>
     <width>384</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NotEurostile</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight></fontweight>
    </font>
   </property>
   <property name="text">
    <string>DOWNLOADING ALICIA.EXE (1/4)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>19</y>
     <width>10</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>19</y>
     <width>13</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_secondary">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>67</y>
     <width>386</width>
     <height>11</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>386</width>
     <height>11</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>386</width>
     <height>11</height>
    </size>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="master.qrc"/>
 </resources>
 <connections/>
</ui>
